### DISCUSSION TITLE
Failed to submit analysis request via Reactome API

### DISCUSSION THREAD

**Discussion by eleonore-schneeg** [reactome/ReactomeGSA#60] [Q&A]
Hi,

I am getting the following error when submitting my request for analysis via the Reactome API. 
My dataset is a large RNAseq count dataset with over 23k genes. How could I solve this error? Does the API support large request like that?

> > > result <- perform_reactome_analysis(request = my_request, verbose = FALSE)
> > Compressing request data...
> > No encoding supplied: defaulting to UTF-8.
> > Error in start_reactome_analysis(request = request, compress = compress,  : 
> >   Failed to submit analysis request: <html>
> > <head><title>413 Request Entity Too Large</title></head>
> > <body>
> > <center><h1>413 Request Entity Too Large</h1></center>
> > <hr><center>nginx</center>
> > </body>
> > </html>
> 
> 

Thanks!

Best wishes,
Eléonore

**Comment by jgriss** [comment]
Hi Eléonore,

Thanks a lot for your interest in ReactomeGSA!

I've increased the max request size on our servers. This hopefully solves your issue.

One thing I recommend though is to filter your genes first based on whether they were observed in a sufficiently high number of samples.

Please keep us posted on your progress!

Kind regards,
Johannes
